decidim-elections/app/controllers/decidim/elections/trustee_zone/application_controller.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Elections
+    module TrusteeZone
+      # This controller is the abstract class from which all trustee_zone
+      # controllers in their public engines should inherit from.
+
+      class ApplicationController < ::Decidim::ApplicationController
+        include Decidim::UserProfile
+
+        helper_method :trustee
+
+        before_action :ensure_configured_bulletin_board!
+
+        private
+
+        def ensure_configured_bulletin_board!
+          return if Decidim::Elections.bulletin_board.configured?
+
+          announcement_args = {
+            callout_class: "alert",
+            announcement: {
+              title: "<strong>#{t("no_bulletin_board.title", scope: "decidim.elections.trustee_zone")}",
```

Comment:
I think we're missing a closing `</strong>` tag here?

Reply:
Ups, yes! I'll fix it.